01-calculate.ipynb

```diff
@@ -9,7 +9,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.        print(f"metrics/{tsv} is up to date with the latest data")
</pre><p>We don't do this for any of the other data sets, and I don't really think it's needed here either.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wikimedia-research/movement-metrics/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>